packages/shared-core/src/sdk/documents/users.ts

```diff
@@ -52,6 +53,18 @@ export function hasAppBuilderPermissions(user?: User | ContextUser): boolean {
   return !isGlobalBuilder && appLength != null && appLength > 0
 }
 
+export function hasAppCreatorPermissions(user?: User | ContextUser): boolean {
+  if (!user) {
+    return false
+  }
+  return _.flow(
+    _.get("roles"),
+    _.values,
+    _.find(x => ["CREATOR", "ADMIN"].includes(x)),
```

Comment:
AppRole literals are defined in a separated module `frontend-core`. Should we duplicate those literals in `shared-core` too?

Reply:
Theoretically speaking can we create direct dependencies between modules in the same project? What's the purpose of `shared-core`? As far as I understand other modules could depend of `shared-core` but `shared-core` must not depend of other modules. So what should we do?

Reply:
Makes sense - in that case I think we are probably ok to just duplicate these literals rather than move things around